sw/device/silicon_creator/lib/sigverify/usage_constraints.h

```diff
@@ -0,0 +1,37 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_SIGVERIFY_USAGE_CONSTRAINTS_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_SIGVERIFY_USAGE_CONSTRAINTS_H_
+
+#include "sw/device/silicon_creator/lib/manifest.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * Gets the usage constraints struct that is used for verifying a ROM_EXT.
```

Comment:
You can also make this stringdoc more generic (if you wish) as this can also be reused in the `ROM_EXT`.

Reply:
SG, done.